BetterShardsBungee/src/main/java/vg/civcraft/mc/bettershards/bungee/BungeeDatabaseHandler.java

```diff
@@ -111,7 +111,7 @@ public ServerInfo getServer(UUID uuid) {
 			ResultSet set = getServer.executeQuery();
 			if (!set.next())
 				return null;
-			return ProxyServer.getInstance().getServerInfo(set.getString(1));
+			return playerServerCache.put(uuid, ProxyServer.getInstance().getServerInfo(set.getString(1)));
```

Comment:
Just FYI -- hashmap.put returns the _prior_ mapping, so in this case, null. Not ideal.